save-demo-cpg/build.gradle.kts

```diff
@@ -19,6 +19,12 @@ repositories {
             artifact("/[organisation].[module]_[revision].[ext]")
         }
     }
+    maven {
+        url = uri("https://s01.oss.sonatype.org/content/repositories/snapshots/")
```

Comment:
its not clear for me
we didn't used snapshot version prevoisly?

Reply:
we used, but removed when migrated to release version. also `save-demo-cpg` has overrides for repositories. that why we need to duplicate configuration here